prometheus-exporters/openstack-exporter/alerts/cinder.alerts

```diff
@@ -29,3 +29,17 @@ groups:
     annotations:
       description: "Cinder backend {{ $labels.shard }}/{{ $labels.backend }} has less than 10% storage left before overcommit is reached."
       summary: "Cinder backend {{ $labels.shard }}/{{ $labels.backend }} has less than 10% storage left before overcommit is reached."
+  - alert: CinderBackendStorageEmptyCritical
+    expr: >
+      sum(cinder_free_capacity_gib == 0) by (region, shard, backend) or sum(cinder_free_capacity_gib == 0) by (region, shard, backend)
```

Comment:
Is this the same condition twice?

Reply:
bah it should be free and total....I'll fix it.